ktlint-ruleset-standard/src/test/kotlin/com/github/shyiko/ktlint/ruleset/standard/ImportOrderingRuleTest.kt

```diff
@@ -21,4 +24,94 @@ class ImportOrderingRuleTest {
             )
         ).isEmpty()
     }
+
+    private val formattedImports =
+        """
+        import android.app.Activity
+        import android.view.View
+        import android.view.ViewGroup
+        import java.util.List
+        import kotlin.concurrent.Thread
+        """.trimIndent()
+
+    private val errorMessage = "Imports must be ordered in lexicographic order in a single group"
```

Comment:
fwiw, tests are good places for redundancy to keep tight readability and little abstraction. i tend to inline most things. don't know about precedent, though, in this project.

Reply:
I'll inline them.

Reply:
f39effaa70b144caa1e41a92f7d71727a6999904